src/dialogs/more-info/more-info-controls.html

```diff
@@ -46,7 +56,7 @@
         icon='mdi:close'
         dialog-dismiss
       ></paper-icon-button>
-      <div main-title>[[_computeStateName(stateObj)]]</div>
+      <div class="main-title" main-title on-click="enlarge">[[_computeStateName(stateObj)]]</div>
```

Comment:
Maybe this would better be suited to be `dblclick` ? That's more in line with expectations from operating systems.

Reply:
Or actually, should it be just a button at the right?

Reply:
Button on the right is taken by "settings"

dblclick doesn't work on mobile and on desktop it selects text.
We could use hammer to listen to double-tap on, but I think it is an overkill and double-tap is not common on mobile.